code/controllers/subsystem/SSdebugview.dm

```diff
@@ -54,13 +54,20 @@ SUBSYSTEM_DEF(debugview)
 	qdel(C.debug_text_overlay)
 
 /obj/screen/debugtextholder
-	icon = 'icons/mob/screen_full.dmi'
-	icon_state = "default"
+	icon = null // just like screentips!
 	screen_loc = "CENTER-7,CENTER-7"
```

Comment:
I know I could just convert this screen_loc to something like `TOP,LEFT` probably, but I don't know much about that. If anyone has any suggestions, I would appreciate it.

Reply:
I set this to `"TOPLEFT"` and it seemed to push all the text to the top left, even on 19x15 screen size. Does that work? 

![image](https://github.com/ParadiseSS13/Paradise/assets/42044220/ac745d49-5c0e-4920-a7e6-7349610f3159)

Reply:
I finally got to messing around with this, couldn't get any of the variations of TOPLEFT or TOP,LEFT to work. I think Im gonna stick with what I have here for now

Reply:
Result of `screen_loc = "TOPLEFT"`:
https://github.com/ParadiseSS13/Paradise/assets/42044220/a57e217b-dec7-489d-8302-bcdc3a1c65a3

Isn't this what we want? (ignoring the Security Assault icon issue of course). For the debug to be stuck in the topleft of the screen. Or am I misunderstanding?

Reply:
Yeah, that is what I want. But it wasn't sticking to the corner like that for me. I can give it another shot

Reply:
Im not sure what I'm doing wrong, but I keep getting *jank* from it. I'll keep at it
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/21e428e8-1ec3-4e61-a7b4-5e3c8edb7baf)

Reply:
Doing some more testing, it only does that because it has that icon, otherwise it does some weird stuff and sends the text off the screen

Reply:
Right, I've managed to get it working using "TOPLEFT" but its a bit more scuffed now because now it uses an empty image to hold the maptext in place, and update_view still needs to be called anyways to update the width of the maptext. I don't know if this is better, but I will leave that up to reviewers. I personally prefer the old way, and can revert if needed.